examples/hybrid-compute/README.md

```diff
@@ -0,0 +1,85 @@
+
+If using Visual Studio code, you can also install the `rust-analyzer` extension in the Extensions tab.
+
+#### Install Rust
+
+Install Rust using the [Rust Installer](https://www.rust-lang.org/tools/install).
+
+**Windows Users:** Before running the installer, you'll need to install the [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/).
+
+#### Create a new Rust project
+
+Navigate to wherever you plan to store your Rust project files, and run: `cargo new --lib rus_wasm`. The project name `rus_wasm` can be whatever project name you want to use.
+
+This creates a new library in a subdirectory with the project name you used with everything you need to get going.
+
+* The `/src/lib.rs` file contains some generated Rust code. This code can be erased and replaced with code you plan to use in your hybrid compute program.
+* Inside this subdirectory is a file called `Cargo.toml`. This file will be used to configure your build.
+
+#### Define classical functions in Rust
+
+Define the classical functions you plan to use in your hybrid compute program in the `src/lib.rs` file. Examples of the types of functions that can be put in Rust code are provided in the `Rust` folders within each of the example folders. *We recommend looking at one of the provided examples to know which functions and attributes are required for Hybrid Compute to work.*
+
+#### Compile code to Wasm
+
+1. Configure the `Cargo.toml` file. You can use one of the examples provided to modify yours. 
+1. Navigate to your Rust project directory from the command line.
+1. Run: `cargo build`. 
+    - If successful, a `target` folder and `Cargo.lock` file will be created, alongside the output of the command displaying "Finished."
+    - If failed, Cargo will output a messaging related to the reason.
+1. Run: `rustup target add wasm32-unknown-unknown`. This allows Rust to used Wasm as a compilation target. This only needs to be run once for your environment. If you have run this previously when compiling Rust to Wasm, you do not need to run it again. 
+1. Run: `cargo build --release --target wasm32-unknown-unknown`. This compiles your Rust code to Wasm.
+
+Congratulations! You've succesfully compiled Rust code to Wasm. 
+
+#### Use compiled Wasm
+
+In order to use the compiled wasm from your quantum program, you will want to find the associated `.wasm` file that the compilation step created. This can be found in the `./target/wasm32-unknown-unknown/release` folder of your project. You'll note a file with the name of your project with the extension `.wasm`. To use this in your quantum program, you can copy-paste it into another folder, or ensure that the file path for your Wasm file handler points to this location.
+
+## Wasm Usage
+
+Some limitations exist on what kinds of functions or capabilities are enabled. 
```

Comment:
I am missing the point of different int types, my understanding is that everything else then i32 will fail? Is that correct?